app/jobs/notifications/remove_each_job.rb

```diff
@@ -0,0 +1,15 @@
+module Notifications
+  class RemoveEachJob < ApplicationJob
+    queue_as :remove_each_notifications
+
+    def perform(notifiable_collection_id, action, service = Notifications::RemoveEach)
```

Comment:
`notifiable_collection_id` is an array of ids so, it should be called `notifiable_collection_ids` to make it clear that the function expects an array

Reply:
Thanks, I have corrected that.